core/tests/webdriverio_utils/ExplorationEditorSettingsTab.js

```diff
@@ -97,36 +100,42 @@ var ExplorationEditorSettingsTab = function() {
       closePreviewSummaryButton);
     await waitFor.invisibilityOf(
       explorationSummaryTile, 'Summary Tile takes too long to disappear');
-    await action.click('Neutral element', neutralElement);
+    await action.click('Neutral element', neutralElement, true);
   };
 
   this.setCategory = async function(category) {
+    await action.click('Neutral element', neutralElement);
     await waitFor.presenceOf(
-      explorationCategoryInput, 'Category input takes too long to be visible.');
+      explorationCategory, 'Category input takes too long to be visible.');
     await (
-      await forms.AutocompleteDropdownEditor(explorationCategoryInput)
+      await forms.AutocompleteDropdownEditor(explorationCategory)
     ).setValue(category);
   };
 
   this.setFirstState = async function(stateName) {
+    await action.click('Neutral element', neutralElement, true);
     await action.waitForAutosave();
     await waitFor.presenceOf(
       initialStateSelect, 'Initial state select takes too long to be visible.');
     await action.click(
       'State name option', initialStateSelectOption(stateName));
+    await action.click('Neutral element', neutralElement);
   };
 
   this.setLanguage = async function(language) {
     await waitFor.presenceOf(
       explorationLanguageInput, 'Language input takes too long to be visible.');
-    await explorationLanguageInput.selectByVisibleText(language);
+    await action.matSelect(
+      'Exploration Language', explorationLanguageInput, language);
+    await action.click('Neutral element', neutralElement);
```

Comment:
Why did we not add `true` for the 3rd argument here?

Reply:
there is no requirement of clicking the variable though js, that's why I have not used true as 3rd argument, + at this point test the dropdown is automatically closed  by the statement.

`
await action.matSelect(
      'Exploration Language', explorationLanguageInput, language);
`